cmd/krel/cmd/patch-announce.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/release/pkg/log"
+	"k8s.io/release/pkg/patch"
+	"k8s.io/release/pkg/util"
+)
+
+// slap the subcommand onto the parent/root
+func init() {
+	cmd := patchAnnounceCommand()
+	rootCmd.AddCommand(cmd)
```

Comment:
init() should be avoided.
the responsibility of the addition should be on the side of the root command.

pseudo-summary of the pattern:
```
parrentCmd := createParentCmd()
childCmd := createChildCmd()
parentCmd.AddCommand(childCmd)
```

Reply:
Yeah, I also don't like the use of `init`, but I also tried to be consistent with what we have right now. If you check my implementation, I did take care of encapsulating my command, not relying on global state or such, so that it could be moved around and the initialisation/registration of the commands could change as needed.

I could change to your proposed registration in this PR, but I lean towards be consistent with what we have right now, and change the registration for all commands in a separate PR.

Reply:
it's just a minor nit and for cobra related usage it should be fine.